I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 123, in script
    call screen instructions_key_screen()
ScriptError: could not find label 'tape'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\User\Documents\renpy\renpy-8.2.1-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\Users\User\Documents\renpy\renpy-8.2.1-sdk\renpy\main.py", line 635, in main
    run(restart)
  File "C:\Users\User\Documents\renpy\renpy-8.2.1-sdk\renpy\main.py", line 145, in run
    renpy.execution.run_context(True)
  File "C:\Users\User\Documents\renpy\renpy-8.2.1-sdk\renpy\execution.py", line 955, in run_context
    context.run()
  File "game/script.rpy", line 123, in script
    call screen instructions_key_screen()
  File "C:\Users\User\Documents\renpy\renpy-8.2.1-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'tape'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.1.24030407
Forensics 1.0
Thu May 30 03:26:04 2024
